addon/fragment.js

```diff
@@ -126,7 +126,12 @@ var Fragment = Model.extend(Ember.Comparable, Ember.Copyable, {
   },
 
   toStringExtension: function() {
-    return 'owner(' + get(internalModelFor(this)._owner, 'id') + ')';
+    let owner = internalModelFor(this)._owner;
+    if (owner) {
+      return 'owner(' + get(owner, 'id') + ')';
+    } else {
+      return '';
+    }
```

Comment:
Was this this a change that resulted from ember/ember-data 2.5 or was it just something to fix while you were in there?